Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -2359,11 +2359,20 @@ def com_google_fonts_check_metadata_canonical_weight_value(font_metadata):
 
 @check(
   id = 'com.google.fonts/check/metadata/os2_weightclass',
-  conditions = ['font_metadata']
+  conditions = ['font_metadata'],
+  misc_metadata = {
+    'request': 'https://github.com/googlefonts/fontbakery/issues/2683'
+  }
 )
 def com_google_fonts_check_metadata_os2_weightclass(ttFont,
                                                     font_metadata):
-  """Checking OS/2 usWeightClass matches weight specified at METADATA.pb."""
+  """Checking OS/2 usWeightClass value matches weight
+  specified at METADATA.pb for static fonts.
```

Comment:
please add 2 extra indentation spaces in these rationale strings otherwise they'll clip on the text terminal reporter

Reply:
Thanks. I thought it would be nice to add a dedicated rationale and to also improve the checks docstring as well.